src/test/resources/balanceComputationParameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "maxNumberIterations" : 11,
+  "thresholdNetPosition" : 2.0,
+  "load-flow-parameters" : {
+    "version" : "1.0",
+    "voltageInitMode" : "DC_VALUES",
+    "transformerVoltageControlOn" : true,
+    "phaseShifterRegulationOn" : true,
+    "noGeneratorReactiveLimits" : false,
+    "specificCompatibility" : false
+  }
+}
```

Comment:
Could you add a new line there? Not sure if the tests fail with the new line, but if it's ok, it's better.

Reply:
the test fail when we add a new line, the roundTripTest method assures that